_layouts/front.html

```diff
@@ -4,174 +4,212 @@
-                    <h3>Mentorship</h3>
-                    <p class="text-muted">In our mentorship program, college undergraduates and high school upperclassmen use their high school experience to guide fellow students to make the most out of their high school experience including essay review, resume building, scholarships, and more.</p>
+        </div>
+    </section>
+
+    <!-- Get Involved! -->
+    <section class="bg-gray" id="get-involved">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h2 class="section-heading">Join Our Team</h2>
+                    <hr class="dark">
+                    <p class="text-muted">
+                        We are always looking for high school and college students to help support our community!
+                        Whether you are good at art, like to post on social media, have great scores on tests and in
+                        school, want to share your college application journey with others, or like writing articles,
+                        there's always a position for you on our team!</p>
+                    <a href="/get-involved" class="btn btn-primary btn-xl">Get Involved!</a>
                 </div>
             </div>
         </div>
-    </div>
-  </section>
-
-  <!-- Get Involved! -->
-  <section class="bg-gray" id="get-involved">
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">Get Involved!</h2>
-                <hr class="dark">
-                <p class="text-muted">
-                    We are always looking for high school and college students to help us support our community! 
-                    Whether you are good at art, like to post on social media, have great scores on tests and in school, 
-                    want to share your college application journey with others, or like writing articles, 
-                    there's always a position for you on our team!</p>
-                <a href="/get-involved" class="btn btn-primary btn-xl">Learn More!</a>
+    </section>
+
+    <!-- Donate! -->
+    <section class="bg-light" id="Donate">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h2 class="section-heading">Support Our Mission</h2>
+                    <hr class="dark">
+                    <p class="text-muted">
+                        As a 501(c)(3) non-profit, EduVisa relies on the generosity of donations to keep
+                        our organization running smoothly and to continue providing our tutoring and
+                        mentorship services free of charge. Each and every donation, no matter how small,
+                        directly helps us reach more students and maintain our worldwide community.
```

Comment:
As a 501(c)(3) non-profit, EduVisa relies on the generosity of donors to keep our organization running smoothly and to continue providing our tutoring and mentorship services free of charge. All donations, no matter how small, directly help us reach more students and maintain our worldwide community. @richardqiu